File: Python/Algorithms/Sorting/insertion_sort.py
def insertion_sort(arr):
   for i in range(1, len(arr)):
      j = i-1
      nxt_element = arr[i]

   while (arr[j] > nxt_element) and (j >= 0):
      arr[j+1] = arr[j]
      j=j-1
   arr[j+1] = nxt_element


if __name__ == "__main__":
  list = [19,2,31,45,30,11,121,27]
  insertion_sort(list)
  print(list)

File: Julia/Algorithms/Sorting/insertionsort.jl
function insertionsort(array)
    for i in 2:length(array)
        current = array[i]
        j = i - 1 
        while j >= 1 && current < array[j]
            array[j + 1] = array[j]
            j -= 1
        end
        array[j+1] = current
    end
end
arr = [rand(-100:100) for i in 1:20]
println(arr)
insertionsort(arr)
println(arr)